src/util/string.h

```diff
@@ -271,7 +271,7 @@ inline bool is_yes(const std::string &str)
 {
 	std::string s2 = lowercase(trim(str));
 
-	return s2 == "y" || s2 == "yes" || s2 == "true" || atoi(s2.c_str()) != 0;
+	return s2 == "y" || s2 == "yes" || s2 == "true" || s2 == "on" || atoi(s2.c_str()) != 0;
```

Comment:
Well, it could be duplicated with a comment notify this - it's not that much code, so wouldn't be that WET. The question is how much is this called in mods? It probably actually isn't that much of a problem as it's mostly used in settings, which should be cached anyway